frontend/e2e/application-noncommercial-group.e2e-spec.ts

```diff
@@ -39,110 +39,93 @@ describe('Apply for a noncommercial group use permit', () => {
-    expect<any>(element(by.id('secondary-permit-holder-first-name')).isPresent()).toBe(false);
-    return checkForSecondPermitHolderAddress(false);
-  });
-  it('should show secondary permit holder name field but hide secondary address fields if add input is checked', () => {
-    element(by.id('add-secondary-permit-holder')).click();
-    expect<any>(element(by.id('secondary-permit-holder-first-name')).isPresent()).toBe(true);
-    return checkForSecondPermitHolderAddress(false);
-  });
-  fieldValidation.validateSimpleTextField('secondary-permit-holder-first-name', 'secondary-permit-holder-first-name-error', 'Secondary permit holder first name is required.');
-  fieldValidation.validateSimpleTextField('secondary-permit-holder-last-name', 'secondary-permit-holder-last-name-error', 'Secondary permit holder last name is required.');
-  it('should show secondary permit holder address fields if "secondary-permit-holder-same-address" is checked', () => {
-    element(by.id('secondary-permit-holder-same-address')).click();
-    return checkForSecondPermitHolderAddress(true);
-  });
-  fieldValidation.validateSimpleTextField('secondary-permit-holder-address', 'secondary-permit-holder-address-error', 'Street address 1 is required.');
-  fieldValidation.validateSimpleTextField('secondary-permit-holder-city', 'secondary-permit-holder-city-error', 'City is required.');
-  fieldValidation.validateSimpleTextField('secondary-permit-holder-state', 'secondary-permit-holder-state-error', 'State is required.');
-  fieldValidation.validateNumberField('secondary-permit-holder-zip', 'secondary-permit-holder-zip-error', 'Valid ZIP code is required.');
-  fieldValidation.validateMinMax('secondary-permit-holder-zip', 'secondary-permit-holder-zip-error', 5, 5);
-  it('should hide secondary permit holder address fields, but remain secondary permit holder name if hide button is clicked', () => {
-    element(by.id('secondary-permit-holder-same-address')).click();
-    expect<any>(element(by.id('secondary-permit-holder-first-name')).isPresent()).toBe(true);
-    return checkForSecondPermitHolderAddress(false);
-  });
-  it('should hide secondary permit holder name and address fields if add secondary input is unchecked', () => {
-    element(by.id('add-secondary-permit-holder')).click();
-    expect<any>(element(by.id('secondary-permit-holder-name')).isPresent()).toBe(false);
-    return checkForSecondPermitHolderAddress(false);
-  });
 
-  fieldValidation.validateSimpleTextField('location', 'location-error', 'Location is required.');
-  fieldValidation.validateNumberField('participants', 'participants-error', 'Participants is required.');
-  fieldValidation.validateSimpleTextField('activity-description', 'activity-description-error', 'Activity description is required.');
+  it('should submit an application as individual with only the required fields populated', () => {
+    page.navigateTo();
+    element(by.id('primary-permit-holder-first-name')).sendKeys('Micky');
```

Comment:
I created an issue to clean up our e2e tests. We can create some better functions here.